### DISCUSSION TITLE
Cover Art for Bella, Bent, Breanna, Brian, Noble and Titan

### DISCUSSION THREAD

**Discussion by bobgeorgethe3rd** [freesewing/freesewing#1626] [Forum]
## Ideas/discussion for the cover art of the body blocks.
The main idea is to use dressform mannequins instead of people to represent the blocks so people know they are blocks rather than patterns.
Below are some ideas I have had for the Blocks.
#### Bella
https://cdn.discordapp.com/attachments/550472363922423808/890003232749940757/20210921_233318.jpg
#### Bent
https://cdn.discordapp.com/attachments/550472363922423808/890003233311961118/20210921_233352.jpg
#### Breanna
https://cdn.discordapp.com/attachments/550472363922423808/890003233035124777/20210921_233345.jpg
#### Brian
https://cdn.discordapp.com/attachments/550472363922423808/890003233567834112/20210921_233357.jpg
#### Noble
Would be the same as the others.
#### Titan
https://cdn.discordapp.com/attachments/550472363922423808/890003233853018132/20210921_233401.jpg

These are all ideas and suggestions on making cover arts for the blocks so the catalouge is complete as it were. Got better ones go for it!

**Comment by darigovresearch** [comment]
Thanks for raising this. Having images for the missing items whether they are an illustration, physical item on a person or a physical item on a mannequin are really helpful to discover what the item is from the catalogue page.

If you have any images of the above items, would highly recommend that you make a pull request. Also this is a good first submission for anyone who wants to contribute if they haven't already!

@bobgeorgethe3rd do you have any images to improve any of the missing items?

**Reply by bobgeorgethe3rd** [reply]
We actually have someone who does the covert art. These are a suggestion on how to represent the blocks in this form.

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
